### PR TITLE
1865 - Fix a few issues with notifications involving styles and structure

### LINKED ISSUE #1573: Notification: Notification cannot be dismissed in Firefox
**Describe the bug**
Notification: Notification cannot be dismissed in Firefox

**To Reproduce**
1. Go to https://4150-beta0-enterprise.demo.design.infor.com/components/notification/example-index.html using Firefox browser
2. Click on 'X' button of the notifications
3. See issue: Notifications were not dismissed

**Expected behavior**
Notification should be dismissed when X button is clicked

**Version**
- ids-enterprise: v4.11.0 to v4.15.0-beta0

**Screenshots**
No screenshots applicable

**Platform**
 - Mac Firefox, Windows Firefox, Android Firefox

**Additional context**
No additional context.

### PR DESCRIPTION
**Pull request by clepore**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
We needed to replace the rgba notification background colors with hexes. This will require `ids-identity@2.5.0` to be released.

Since I was under the hood, I also updated the structure to fix #1573 and made the responsive view more "flexible".
- `html` - changed the`<span>` to a `<p>` for wrapping
- `css` - removed `line-height` style in favor of padding around the whole thing to allow text wrapping
- `css` - updated tokens with specific notification tokens
- `css` - removed conflicting positioning of icons
- `js` - separated the js code into smaller string snippets
- `js` - removed the string with interpolation of an if-statement with inside strings with interpolation
- `js` - included the`<a>` inside of the `<p>` tag for semantics and proper wrapping
- `demo`- added a long notification to test wrapping

**Related github/jira issue (required)**:
- Fixes https://github.com/infor-design/enterprise/issues/1857
- Fixes https://github.com/infor-design/enterprise/issues/1865
- Fixes https://github.com/infor-design/enterprise/issues/1573

**Steps necessary to review your pull request (required)**:
1. 📝  `npm i && npm run start`
1. 📝 Testing #1857 and #1865:
    - Go to [localhost notifications](http://localhost:4000/components/notification/example-index.html?layout=embedded)
    - verify that the colors match up to the description in [this comment](https://github.com/infor-design/design-system/issues/294#issuecomment-476274897) and in [this pull request](https://github.com/infor-design/design-system/pull/371)
    - verify the colors are "bolder" than [4.17.1](https://4171-enterprise.demo.design.infor.com/components/notification/example-index.html)
    - verify the responsive views wrap text an visually look up to par (no overlapping of text/icons, etc)
1. 📝 Testing #1573
    - In Firefox, go to [localhost notifications](http://localhost:4000/components/notification/example-index.html) 
    - verify you can click the close `x` button
    - bonus is that resizing the browser will adjust for long notifications w/o an ellipsis ([compare to current responsive here]([4.17.1](https://4171-enterprise.demo.design.infor.com/components/notification/example-index.html))

ToDo:
☑️  A note to the change log
☑️  Need `ids-identity@2.5.0` to be released

### CODE REVIEW

**Comment by tmcconechy** [comment]
This branch is stuck, maybe needs a push to get it going. You could just add an entry to the CHANGELOG.MD to do that

**Comment by clepore** [comment]
@tmcconechy We have no released ids-identity@2.5.0 yet. This PR is still in draft.

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 - `[Listbuilder]` Fixed an issue where the text was not sanitizing. ([#1692](https://github.com/infor-design/enterprise/issues/1692))
 - `[Locale]` Fixed trailing zeros were getting ignored when displaying thousands values. ([#404](https://github.com/infor-design/enterprise/issues/1840))
 - `[MenuButton]` Approved the way menu buttons work with screen readers.([#404](https://github.com/infor-design/enterprise/issues/404))
+- `[Notifications]` Fixed a few issues with notificationb background colors by using the corresponding ids-identity token for each. ([1857](https://github.com/infor-design/enterprise/issues/1857), [1865](https://github.com/infor-design/enterprise/issues/1865))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md]:
```suggestion
- `[Notifications]` Fixed a few issues with notification background colors by using the corresponding ids-identity token for each. ([1857](https://github.com/infor-design/enterprise/issues/1857), [1865](https://github.com/infor-design/enterprise/issues/1865))
```

Comment (clepore) [reply docs/CHANGELOG.md]: Good 👁

**Comment by clepore** [comment]
@tmcconechy I also fixed #1573 per your comment. It made sense since I was in there. I upated the PR description/testing instructions to reflect those changes.

**Comment by tmcconechy** [comment]
@clepore we have a conflict here but im not sure i can resolve it with the button

File: app/views/components/notification/example-index.html [hunk 2]
```diff
@@ -1,44 +1,44 @@
...
   });
 </script>
 
 <script>
-  $('#fake-header').notification({
+  $('#mock-header').notification({
     type: 'info',
-    message: 'DTO rejcted by your manager for Sept 30, 2018.',
-    parent: '#fake-header',
+    message: 'DTO rejcted by your manager for Sept 30, 2018. Please login to your Human Resources (HR) portal for an explanation on why. If you believe there was a mistake or wish to appeal the rejection, you can now also do that in the new HR portal. ',
```
Comment (clepore) [inline app/views/components/notification/example-index.html]:
```suggestion
    message: 'DTO rejected by your manager for Sept 30, 2018. Please login to your Human Resources (HR) portal for an explanation on why. If you believe there was a mistake or wish to appeal the rejection, you can now also do that in the new HR portal. ',
```

**Pull request merged**